tests/s-arg_oct.c

```diff
@@ -0,0 +1,12 @@
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+int main()
```

Comment:
I think valid C requires `void` in the argument.

Reply:
Added it. :)